Add instructions for bypassing SSL verification (#1940)
grafana/k6-docs: docs/sources/k6-studio/troubleshoot.md

Some proxies or HTTPS sites may require a certificate to establish a trusted connection. In this case, specify the certificate in the "Certificate path" field.

{{< admonition type="caution" >}}

> You can bypass SSL/TLS validation in trusted environments by checking the "Skip SSL/TLS validation" option in k6 Studio settings. Use it carefully as it leaves connections open to man-in-the-middle (MITM) attacks.

Reviewer: Small nit, sorry I missed this! We use bold when referring to UI elements.

```suggestion
You can bypass SSL/TLS validation in trusted environments by checking the **Skip SSL/TLS validation** option in k6 Studio settings. Use it carefully as it leaves connections open to man-in-the-middle (MITM) attacks.
```

Some proxies or HTTPS sites may require a certificate to establish a trusted connection. In this case, specify the certificate in the "Certificate path" field.

> Optionally, you can bypass SSL/TLS validation in trusted environments by checking the "Skip SSL/TLS validation" option in k6 Studio settings. This introduces vulnerabilities to TLS interception, please use it carefully.

Reviewer: ```suggestion
{{< admonition type="caution" >}}

You can bypass SSL/TLS validation in trusted environments by checking the "Skip SSL/TLS validation" option in k6 Studio settings. Use it carefully as it introduces vulnerabilities to TLS interception.

{{< /admonition >}}
```

I think we can use the [`caution` admonition](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#cautions) here since there are some security considerations. What do you think?

Reviewer 2: > Use it carefully as it introduces vulnerabilities to TLS interception.

This reads a bit "weird" to me. Does the TLS interception get vulnerable? Or is something vulnerable to being intercepted? I would suggest adding a bit more context:

> Use it carefully as it leaves connections open to TLS interception.

And TLS interception isn't necessarily a bad thing (in fact TLS interception is a feature of k6 Studio). So we could be even more explict and list the exact danger so that it's easier for users to read up on it:

> Use it carefully as it leaves connections open to Man-in-the-middle attacks.

What do you think?

Author: I love the suggestion, thanks both! 🙏
